test/functional/fixtures/readfile.txt

```diff
@@ -0,0 +1,4 @@
+a
+
+
+	a
```

Comment:
Usually we use `write_file()` to write temporary files in tests (then clean them up in `after_each`). Is there a reason that wouldn't work for this?